custom/icds_core/const.py

```diff
@@ -7,3 +7,4 @@
 LocationReassignmentView_urlname = 'location_reassignment'
 ICDS_DOMAIN = 'icds-cas'
 IS_ICDS_ENVIRONMENT = settings.SERVER_ENVIRONMENT in settings.ICDS_ENVS
+CPMU_ROLE_NAME = "CPMU"
```

Comment:
this const is not needed here right?

Reply:
true, but then it needs to be moved to the ICDS repo